### ISSUE TITLE
Add FIDO2/Yubikey/WebAuthn authentication support

### ISSUE THREAD

**Issue by newbold** [neatnik/omg.lol#44]

**Comment by maique** [comment]
Cool! I have a YubiKey arriving right now, slowly, and it’s great to see I’ll be able to use it here as well!

**Comment by lakerjv** [comment]
i agree with this!! ive been a yubikey user for abt 4 months now i thing and security is yes :)

**Comment by newbold** [comment]
This item is being tracked in the new omg.lol Feedback Forum. To follow its status, visit: https://feedback.omg.lol/1660512170215

**Comment by newbold** [comment]
This is prerequisite for #310.

**Comment by pawelorzech** [comment]
+1 from me then since I'm all on these yubikeys:)

**Comment by byemc** [comment]
Feel like @newbold  should close this as it looks like this is all implemented

**Comment by newbold** [comment]
@byemc well, the one remaining piece is adding it as 2FA, with or in lieu of an authenticator. Right now the only limited support is with a passkey, so it’s not yet _technically_ complete. 🤓
